qa/vagrant/src/test/resources/packaging/scripts/70_sysv_initd.bats

```diff
@@ -117,6 +117,17 @@ setup() {
     [ "$status" -eq 3 ] || [ "$status" -eq 4 ]
 }
 
+@test "Don't mkdir when it contains a comma" {
+    sed -i 's/DATA_DIR=.*/DATA_DIR=\/tmp\/foo,\/tmp\/bar/' /etc/init.d/elasticsearch
+    service elasticsearch start
+    wait_for_elasticsearch_status
+    assert_file_not_exist /tmp/foo,/tmp/bar
+    service elasticsearch stop
+    run service elasticsearch status
+    # precise returns 4, trusty 3
+    [ "$status" -eq 3 ] || [ "$status" -eq 4 ]
```

Comment:
> But do we understand the underlying source of the difference?

I have no idea what the source of the difference is (I briefly looked and `service` is supposed to pass through the exit code). I can look into it, maybe in a different issue though?